POO/Clase 24 - Herencia/telefono.ts

```diff
@@ -0,0 +1,49 @@
+class Telefono{
+    private estaPrendido: boolean;
+    protected bateriaActual:number;
+
+    public constructor(){
+        this.estaPrendido = false;
+        this.bateriaActual= 100;
+    }
+    public prenderApagar():void{
+        if (!this.estaPrendido)
+            this.estaPrendido = true;
+        else
+            this.estaPrendido = false;
```

Comment:
Toda esta funcion la podrias simplificar con una sola linea haciendo `this.estaPrendido = !this.estaPrendido` pero esta bien como lo hiciste, es solo una recomendacion para que quede mejor :)